internal/handlers/tigris/tigrisdb/query.go

```diff
@@ -95,25 +110,45 @@ func (tdb *TigrisDB) BuildFilter(filter *types.Document) driver.Filter {
 	res := map[string]any{}
 
 	for k, v := range filter.Map() {
-		// filter only by _id for now
-		if k != "_id" {
+		// don't pushdown empty strings yet
```

Comment:
For those cases, I believe that the empty string filter is the last one in terms of incompatibility. They work currently on API, but not on driver. I assume it'll be shortly fixed so I'll cover that on dotnotation task.

Reply:
@noisersup let's create a separate issue and add a TODO here?